### DISCUSSION TITLE
"show all assets" for a specific page doesn't work

### DISCUSSION THREAD

**Discussion by PikuZheng** [community/community#41339] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

[On this page](https://github.com/pymumu/smartdns/releases ), when I click the show all 28 assets link below release 39, it gets stuck at loading.

![image](https://user-images.githubusercontent.com/12579522/206856270-222d852c-01c8-45d5-a058-664716211f63.png)

Strangely I didn't find similar issues on other pages. Chrome debugging shows that when I click the link, the page does not initiate xhr. I would like to know what is triggering this issue.

**Comment by jfullstackdev** [comment, marked as answer]
try visiting the full page, it was loading when I clicked it too but when I opened it as one page, it did not

![image](https://user-images.githubusercontent.com/47092464/206865746-a9d3c819-51c3-4a81-ac8c-2e966418af90.png)

this is the link 

https://github.com/pymumu/smartdns/releases/tag/Release39

**Reply by PikuZheng** [reply]
yes it is but why😂

**Reply by jfullstackdev** [reply]
maybe it's a bug in JavaScript that it cannot load anymore at that page, but with the solo page for that release it's just fine

in actual development, we can't write a perfect code, so it's always a trade-off what's important and not so important
